tests/unit/test_bugzilla.py

```diff
@@ -31,10 +31,16 @@ def test_extract_see_also(see_also, expected):
 @pytest.mark.parametrize(
     "whiteboard",
     [
+        "[DevTest-]",
+        "[-DevTest-]",
+        "[-DevTest]",
+        "[DevTest-test]",
+        "[test-DevTest]",
+        "[foo-DevTest-bar]",
+        "[foo-bar-DevTest-foo-bar]",
         "[example][DevTest]",
-        "[example][DevTest-test]",
-        "[example][test-DevTest]",
-        "[example][foo-DevTest-bar]",
+        "[DevTest][example]",
+        "[example][DevTest][example]",
     ],
```

Comment:
Should we also add test cases for whiteboard strings that include space delimiters as well?